lib/guardian_db/token/sweeper_server.ex

```diff
@@ -0,0 +1,57 @@
+defmodule GuardianDb.Token.SweeperServer do
+  @moduledoc """
+  Periocially purges expired tokens from the DB.
+
+  ## Example
+    config :guardian_db, GuardianDb,
+      sweep_interval: 60 # 1 hour
+
+    # in your supervisor
+      worker(GuardianDb.Token.SweeperServer, [])
+  """
+
+  use GenServer
+  alias GuardianDb.Token.Sweeper
+
+  def start_link(opts \\ []) do
+    defaults = %{
+      interval: Sweeper.get_interval()
```

Comment:
@doomspork I promote the interval calculation to the genserver state so if we need to add more workers in the future we can just pass `interval: ...` throw the opts